playbooks/roles/nginx/defaults/main.yml

```diff
@@ -24,6 +24,9 @@ NGINX_USERS:
     password: "{{ COMMON_HTPASSWD_PASS }}"
     state: present
 
+NGINX_HEALTH_CHECK_ENABLED: False
+NGINX_HEALTH_CHECK_USER_AGENT: ""
```

Comment:
This will be set in the server vars to match the user agent of the health checker.